lib/data_migrate/tasks/data_migrate_tasks.rb

```diff
@@ -16,7 +20,7 @@ def migrate
         if Rails::VERSION::MAJOR == 5 && Rails::VERSION::MINOR == 2
           DataMigrate::MigrationContext.new(migrations_paths).migrate(target_version)
         else
-          paths = migrations_paths || "db/data/"
+          paths = migrations_paths
```

Comment:
Isn't this better if you juts change the line below with `migration_paths`? My guess is that they have that line because of the `||` but now it is not there.
```ruby
else 
  DataMigrate::DataMigrator.migrate(migration_paths, ENV["VERSION"] ? ENV["VERSION"].to_i : nil)
end
```

Reply:
sounds good